lib/material/extra/calendar/calendar.flow

```diff
@@ -928,15 +929,16 @@ dayEventsPopup(
 	close = make(false);
 	dayEvents = make(dayEventsStatic);
 
-	curTime = stamp2time(timestamp());
```

Comment:
Looks like it will break current behaviour. Currently if you add new event the time would be rounded current time which we get from timestamp(). But if you use new curTime it will be current date with time 0:00:00. So default event time will always be rounded 0:00:00, instead of rounded current time.

Reply:
what about using this kind of function instead of using date2time?

date2currentTime(date : Date) -> Time {
  Time(getCurrentTime() with year = date.year, month = date.month, day = date.day)
}